libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/RoadObjectsStore.kt

```diff
@@ -40,7 +45,7 @@ class RoadObjectsStore internal constructor(
      */
     fun getRoadObjectMetadata(roadObjectId: String): EHorizonObjectMetadata? {
         return navigator.roadObjectsStore?.getRoadObjectMetadata(roadObjectId)?.let {
-            navigator.navigatorMapper.run {
+            run {
```

Comment:
Do we need this `run` at all? Do you remember why was added in the first place @korshaknn?